_includes/snippets_library/UCL_Myriad_sge/12/info.snip

```diff
@@ -0,0 +1,14 @@
+```
+Unknown node type: util02
+Unknown node type: util05
+    2 type * nodes: 36 cores, 188.4G RAM
+    7 type B nodes: 36 cores,   1.5T RAM
+   66 type D nodes: 36 cores, 188.4G RAM
+    9 type E nodes: 36 cores, 188.4G RAM
+    1 type F nodes: 36 cores, 188.4G RAM
+    3 type H nodes: 36 cores, 172.7G RAM
+   53 type H nodes: 36 cores, 188.4G RAM
+    3 type I nodes: 36 cores,   1.5T RAM
+    2 type J nodes: 36 cores, 188.4G RAM
+```
+{: .output}
```

Comment:
POSIX requires a newline at the end of the file.

Reply:
Yeah, all these lines are corrected on #156 (f5e136d31c500d0a652b8e5afb1c29b35de92536 to be exact) and this will be rebase once that's merged.

Reply:
:+1: